config.yaml

```diff
@@ -65,18 +59,16 @@ teams:
   - name: besu-triage
     maintainers:
       - macfarla
-      - non-fungible-nelson
+      - jflo
+      - jframe
     members:
+      - ajsutton
+      - antonydenyer
+      - atoulme
       - NickSneo
-      - cdivitotawela
-      - cloudspores
-      - ensi321
       - gtebrean
       - kkaur01
-      - lu-pinto
-      - m4sterbunny
-      - Matilda-Clerke
-      - matthew1001
+      - shemnon
```

Comment:
```suggestion
```

Reply:
this is the triage group, which gives limited access only (I think you can assign an issue and add labels) - this is where I've put other emeritus maintainers. But can remove you from here too if you'd prefer @shemnon

Reply:
I want no role in LFDT governance, to make sure expectations for my participation are at an appropriate level.

Reply:
understood @shemnon